src/pudl/metadata/resources/glue.py

```diff
@@ -2,39 +2,20 @@
 from typing import Any
 
 RESOURCE_METADATA: dict[str, dict[str, Any]] = {
-    "assn_gen_eia_unit_epa": {
+    "epacamd_eia": {
         "schema": {
             "fields": [
```

Comment:
This file is very similar to the `metadata/resources/pudl.py` file. Both have the same doc strings header: `Definitions for the glue/crosswalk tables that connect data groups.` 

The `pudl.py` file has metadata for pudl-specific tables: `plants_pudl`, `utilities_pudl`, `utility_plant_assn` and the `glue.py` file only has the `epacamd_eia` table. Should we merge them or keep them seperate?

There are some differences (pudl vs non-pudl glue), so if we decide to keep them distinct that's fine, but I'd like to clarify the doc strings so they at least say something different.

Reply:
The stuff in `pudl.py` is limited to the manual plant and utility mapping that we do -- it's about connections between our PUDL specific IDs and other datasets, which sits pretty deep down inside the DB.

The stuff in `glue.py` is about connections between other datasets that we're bringing in.

It sounds like both docstrings are too short, and the one in `pudl.py` should be different.

Reply:
@zaneselvans

Reply:
Ok I updated the pudl one so it's different. Not going to go into too much detail rn